chat-react-app/src/App.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+
+function App() {
+  const [messages, setMessages] = useState([]);
+  const newMessage = { from: "", text: "" };
+
+  useEffect(() => {
+    getMessages();
+    const interval = setInterval(() => getMessages(), 10000);
+    return () => clearInterval(interval);
+  }, []);
```

Comment:
Great job! Efficiently using the `useEffect` and `clearInterval`

Reply:
I am planning to use websocket to achieve real time communication. I'll try to implement it today. Can we have a meeting about websockets if you have time? I quite didn't understand it 😆